sound/hybrid_font.h

```diff
@@ -120,7 +130,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      } else {
+        second_swing_player_ = PlayPolyphonic(&swng);
+      }
+    } else {
+      PlayMonophonic(&swing, &hum);
+    }
+  }
+
+  float SetSwingVolume(float swing_strength, float mixhum) {
+    if (IsSwingPlaying()) {
+      float accent_volume = powf(swing_strength, config_.ProffieOSSwingVolumeSharpness) * config_.ProffieOSMaxSwingVolume;
+      swing_player_->set_volume(accent_volume);
+      if (second_swing_player_) {
+        second_swing_player_->set_volume(accent_volume);
+      }
+      mixhum = mixhum * config_.ProffieOSMaxSwingDucking;
+    }
+    return mixhum;
+  }
 
+  bool IsSwingPlaying() {
+    if (swing_player_) {
+      if (second_swing_player_) {
+        if(second_swing_player_->isPlaying() || swing_player_->isPlaying()) {
+          return true;
+        }
+      } else {
+        if (!swing_player_) {
```

Comment:
This is wrong.
We should free the swing_player_ if it is true but not playing.
Same for the second_swing_player_.

Reply:
fixed.

Reply:
removed second_swing_player